webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -1,82 +1,52 @@
-  const [activeAccordion, setActiveAccordion] = useState(0);
+const CardItem = ({
+  header,
+  children,
+  dropdownItems,
+  overrideGridProps,
+  overrideDropdownProps,
+}) => {
+  const [dropdownVisibility, setDropdownVisibility] = useState(false);
+  const onDropdownToggle = isOpen => setDropdownVisibility(isOpen);
 
-  const onToggle = id => {
-    if (id === activeAccordion) {
-      setActiveAccordion('');
-    } else {
-      setActiveAccordion(id);
-    }
+  const onDropdownSelect = event => {
+    setDropdownVisibility(prevState => !prevState);
+    // https://github.com/eslint/eslint/issues/12822
+    // eslint-disable-next-line no-unused-expressions
+    overrideDropdownProps?.onSelect?.(event);
   };
-
   return (
     <GridItem xl2={3} md={6} lg={5} {...overrideGridProps}>
       <Card isHoverable>
-        <CardTitle>{header}</CardTitle>
-        <CardBody>
-          {children || (
-            <Accordion asDefinitionList>
-              {!content.length && (
-                <div style={{ marginLeft: '20px' }}>
-                  <Skeleton count={3} width={200} />
-                </div>
-              )}
-
-              {content.map(({ id, name, key, value, href }) => (
-                <AccordionItem key={id}>
-                  <AccordionToggle
-                    onClick={() => {
-                      onToggle(id);
-                    }}
-                    isExpanded={activeAccordion === id}
-                    id={id}
-                  >
-                    {name}
-                  </AccordionToggle>
-                  <AccordionContent id={id} isHidden={activeAccordion !== id}>
-                    <DataList aria-label="Parameters" isCompact>
-                      <DataListItem aria-labelledby="Parameter's type">
-                        <DataListItemRow>
-                          <DataListItemCells
-                            dataListCells={[
-                              <DataListCell key={`${name}-type`}>
-                                <span>
-                                  {href ? <a href={href}>{key}</a> : key}
-                                </span>
-                              </DataListCell>,
-                              <DataListCell key={`${name}-type-content`}>
-                                {value}
-                              </DataListCell>,
-                            ]}
-                          />
-                        </DataListItemRow>
-                      </DataListItem>
-                    </DataList>
-                  </AccordionContent>
-                </AccordionItem>
-              ))}
-            </Accordion>
+        <CardHeader>
+          {dropdownItems && (
+            <CardActions>
+              <Dropdown
+                onSelect={onDropdownSelect}
+                toggle={<KebabToggle onToggle={onDropdownToggle} />}
+                isOpen={dropdownVisibility}
+                isPlain
+                dropdownItems={dropdownItems}
```

Comment:
An understanding question -looks like you can also pass the items as children (which is a bit more readable at least to me):
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Dropdown/DropdownItem.tsx#L61
what is the difference between the two implementations (other than one using `<ul>` and the other `<div>` and missing `aria-popover`)? and why is one better than the other?

Reply:
I agree using children is a bit more readable but it looks odd : 
![Screen Shot 2021-07-05 at 20 08 23](https://user-images.githubusercontent.com/11807069/124503395-22190a80-ddce-11eb-9f96-07934c3e0c6a.png)